2.6.md

```diff
@@ -2294,6 +2299,16 @@ This object provides information pertaining to the device through which the user
     <td><code>macmd5</code></td>
     <td>string; DEPRECATED</td>
     <td>MAC address of the device; hashed via MD5.</td>
+ <tr>
+   </tr>
+    <td><code>brand</code></td>
```

Comment:
Is this not the same as Device.make? If not, how is this different?

Reply:
I see – this can group multiple models of the same device manufacturer into a single device brand (such as Galaxy S-series). But is this not redundant with a combination of Device.make + Device.model, then? How valuable would it be to add such a somewhat redundant grouping into a bid request?

Reply:
It's in use widely by Chinese ad networks today, where sub-brands are quite separate, and each sub-brand is targeted at different demographics. For example, Xiaomi operates the sub-brands Poco and Redmi. As each sub-brand has upwards of 30 devices, using make and model would be onerous. In addition, networks use this to implement basic anti-fraud techniques, matching models with correct sub-brands.